src/integration/java/org/springframework/data/couchbase/repository/typealias/TypeAliasEntityRepositoryTests.java

```diff
@@ -0,0 +1,134 @@
+/*
+ * Copyright 2017 the original author or authors.
```

Comment:
This (and the other) integration tests need to move to src/test/...  There are other integration tests there, let's move these, add the annotations `@RunWith` and `@ContextConfiguration`, etc..